.github/workflows/build_flang_arm64.yml

```diff
@@ -0,0 +1,96 @@
+name: Flang build & test ARM64
+
+on:
+  push:
+    branches: [ master ]
+    paths-ignore:
+      - '**/.github/workflows/build_flang.yml'
+  pull_request:
+    branches: [ master ]
+    paths-ignore:
+      - '**/.github/workflows/build_flang.yml'
+
+jobs:
+  build_flang:
+    runs-on: self-hosted
+    env:
+      build_path: /home/github
+      install_prefix: /home/github/usr/local
+    container:
+      image: ghcr.io/${{ github.repository_owner}}/ubuntu20-flang-${{ matrix.llvm_branch }}:latest
+      credentials:
+        username: github
+    strategy:
+      matrix:
+        target: [AArch64]
+        cc: [gcc]
+        cpp: [g++]
+        version: [10]
+        llvm_branch: [release_11x, release_12x]
```

Comment:
Perhaps this was discussed already in #1044 or even earlier but are we leaving out `release_100` intentionally?

Reply:
Yes, I only mentioned this in the technical call on the 21st of April ([see meeting minutes](https://docs.google.com/document/d/1-OuiKx4d7O6eLEJDBDKSRnSiUO2rgRR-c2Ga4AkrzOI/edit#)):

> Due to limited resources the builds would primarily run for release_11x and release_12x with gcc-10 with the possibility of further expansion if more runners are added or if the load proves to be low enough

We have very fast 32-core servers, but just two of them at the moment. Before we add more than two jobs we wanted to make sure that ARM64 builds do not become a bottle neck for PRs. I remember Huawei is most interested in even-number llvm releases, so please align with @kiranchandramohan  and @RichBarton-Arm if adding release_100 is crucial for you. It's just a matter of risking that sometimes we'll have to queue jobs before the runners are available.

Reply:
Oh right, I remember that discussion. Thanks.